server/src/ClientToDBAPI/CallRoute/mainRoute.ts

```diff
@@ -0,0 +1,18 @@
+import { Router, Request, Response } from 'express';
+import axios from 'axios';
+const callCenterRouter = Router();
+
+callCenterRouter.get('/:phoneNumber/:agentId', (request: Request, response: Response) => {
+    const { phoneNumber, agentId } = request.params;
+    const click2dialURL = "http://10.99.83.19:5010/AceHomeWS/Services.asmx/Click2dial?PhoneNumber="
```

Comment:
Maybe put it inside .env

Reply:
We talked about it. It will come up in the next MVP